### DISCUSSION TITLE
Creating a central dashboard for reviewers(?)

### DISCUSSION THREAD

**Discussion by harshtandiya** [fossunited/fossunited#416] [Ideas]
Currently the reviewers have to go to specific events, and go through the different CFPs and leave their review on the CFP Submission page.

My proposal is that we create a dashboard for reviewers. In this dashboard, the events that need reviews will be listed down. The reviewer can select that event, and then the CFPs will be available in a list view. One would be able to click on the CFP, see all their details, leave their remarks and mark it as accepted / rejected.

The idea is to have all the CFPs at a central location + bringing better reactivity with VueJS as opposed to HTML JS Jinja that is there currently.

**Comment by shhivam** [comment]
I think it would be helpful to see all the events(which have a CFP requirement) in a central dashboard.

> bringing better reactivity with VueJS as opposed to HTML JS Jinja that is there currently.

My role is limited to CFP reviewer so I might not have all the context around the tech decisions.
However, I am curious to understand more about this. Can you elaborate on some examples around improving reactivity?

**Comment by harshtandiya** [comment]
By improving reactivity, I mean that a proper list in vue will enable other features such as filter, group etc

**Reply by shhivam** [reply]
Ah okay, understood!

**Comment by SphericalKat** [comment]
This sounds reasonable. I'm assuming the reviews will still be separated by event for clarity. 

One feature I hope this will have is to filter out already reviewed proposals. (Indeed it sounds like filtering is a thing from the above conversation)
